eclide/BookmarksView.cpp

```diff
@@ -506,7 +513,7 @@ void CBookmarksView::ParseBookmarksEcl(const std::_tstring & ecl, const std::_ts
                         if (boost::algorithm::iequals(liner,nStr) && boost::algorithm::iequals(module,inModule) && boost::algorithm::iequals(attributeName,inAttributeName))
                         {
                             std::_tstring inComment = m_list.GetItemText(i, 6);
-                            if (boost::algorithm::iequals(comment,inComment))
+                            if (!boost::algorithm::iequals(comment,inComment))
```

Comment:
Is this correct - not sure what is being checked?

Reply:
This is correct. This is checking to see if the description in the bookmark is different. If so, it updates the display.